sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -539,6 +539,11 @@ public void removeExtra(final @NotNull String key) {
     return NoOpScopesStorage.NoOpScopesLifecycleToken.getInstance();
   }
 
+  @Override
+  public @NotNull ISentryLifecycleToken pushIsolationScope() {
+    return NoOpScopesStorage.NoOpScopesLifecycleToken.getInstance();
```

Comment:
Shouldn't this also be `Sentry.pushIsolationScope()`? Same as in HubAdapter?

Reply:
`Hub` should be removed before releasing the major IMO.